pkg/subctl/cmd/version/version.go

```diff
@@ -44,3 +54,36 @@ func subctlVersion(cmd *cobra.Command, args []string) {
 func PrintSubctlVersion(w io.Writer) {
 	fmt.Fprintf(w, "subctl version: %s\n", version.Version)
 }
+
+func CheckRequirements(config *rest.Config) (string, []string, error) {
+	failedRequirements := []string{}
```

Comment:
No need for this to be a list.. It's either no values or one..

Reply:
yeah the way it's currently implemented but I simply moved this as is so if we're to change the API then I think that should be done in a separate PR. I rather not make this one any larger than it already is 😄